mattermost-community/mattermost_aws.yaml

```diff
@@ -0,0 +1,50 @@
+name: mattermost-community
```

Comment:
I wonder if, for simplicity, we should just call it `mattermost`, it should not be too different to deploy another version right?

Reply:
Actually, it is-- the helm charts are very different